drivers/include/mtd_spi_nor.h

```diff
@@ -12,8 +12,33 @@
  * @ingroup     drivers_storage
  * @brief       Driver for serial NOR flash memory technology devices attached via SPI
  *
- * @{
  *
+ * @section mtd_spi_nor_overview Overview
+ * Various manufacturers such as ISSI or Macronix offer small SPI NOR Flash which usually
+ * come in 8-pin packages and are used for persistent data storage.
+ * This driver adds support for these devices with support for RIOT's MTD subsystem.
+ *
+ * @section mtd_spi_nor_usage Usage
+ *
+ * The basic functions of all flash devices can be used by just including the
+ * mtd_spi_nor module.
+ *
+ *      USEMODULE += mtd_spi_nor
+ *
+ * For ISSI and Macronix devices, some security features are provided
```

Comment:
s/security/integrity ?

Reply:
When reading 'Security' I tend to think about encryption and prevention from unauthorized access, not protection against data corruption.